assets/js/view/spreadsheet.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+module View
+  class Spreadsheet < Snabberb::Component
+    needs :game
+
+    def render
+      h(:div, { style: {
+        overflow: 'auto',
+        margin: '0 -1rem'
+      } }, [render_table])
+    end
+
+    def render_table
+      h(:table, { style: {
+        margin: '1rem 0 1.5rem 0',
+        'text-align': 'center',
+      } }, [
+        *render_title,
+        *render_corporations,
+        h(:tr, [
+          h(:td, { style: { width: '20px' } }, ''),
+          h(:th, { attrs: { colspan: @game.players.length } }, 'Player Finances'),
+        ]),
+        render_player_cash,
+        render_player_privates,
+        render_player_worth,
+        render_player_certs,
+      ])
+      # TODO: consider adding OR information (could do both corporation OR revenue and player change in value)
+      # TODO: consider adding train availability
+    end
+
+    def render_title
+      [
+        h(:tr, [
+          h(:th, { style: { width: '20px' } }, ''),
+          h(:th, { attrs: { colspan: @game.players.length } }, 'Players'),
+          h(:th, { attrs: { colspan: 2 } }, 'Bank shares'),
+          h(:th, { attrs: { colspan: 2 } }, 'Prices'),
+          h(:th, { attrs: { colspan: 4 } }, 'Corporation'),
+          h(:th, { style: { width: '20px' } }, ''),
+          ]),
+        h(:tr, [
+          h(:th, { style: { width: '20px' } }, ''),
+          *@game.players.map { |p| h(:th, p.name) },
+          h(:th, 'IPO'),
+          h(:th, 'Market'),
+          h(:th, 'Par'),
+          h(:th, 'Current'),
+          h(:th, 'Cash'),
+          h(:th, 'Trains'),
+          h(:th, 'Tokens'),
+          h(:th, 'Privates'),
+          h(:th, { style: { width: '20px' } }, ''),
+        ])
+      ]
+    end
+
+    def render_corporations
+      @game.corporations.map { |c| render_corporation(c) }
+    end
+
+    def render_corporation(corporation)
+      corporation_color =
+        {
+          style: {
+            background: corporation.color,
+            color: '#ffffff'
+          }
+        }
+      props = { style: {} }
+      props[:style]['background-color'] = 'rgba(220,220,220,0.4)' unless corporation.floated?
```

Comment:
It'd be nice to add yellow, brown colour for when the shares are in that area. But that's future work.